api/src/task/opportunities/export_opportunity_data_task.py

```diff
@@ -119,8 +145,18 @@ def export_data_to_json(self, data_to_export: dict) -> None:
         with file_util.open_stream(self.json_file, "w") as outfile:
             outfile.write(json_object)
 
-    def export_opportunities_to_csv(self, opportunities: Sequence[dict]) -> None:
+        return len(json_object.encode("utf-8"))
+
+    def export_opportunities_to_csv(self, opportunities: Sequence[dict]) -> int:
         # create the csv file
         logger.info("Creating Opportunity CSV extract", extra={"csv_extract_path": self.csv_file})
+
+        csv_buffer = StringIO()
+        opportunities_to_csv(opportunities, csv_buffer)
+        csv_data = csv_buffer.getvalue()
+        csv_size = len(csv_data.encode("utf-8"))
```

Comment:
Rather than effectively make the file twice, what if we just ask the file system/s3.

We'd need to do that ourselves, but a method in our file_utils like (hastily thrown together from copying a few different bits of code):

```
def get_file_length_bytes(path: str) -> int:
     if is_s3_path(path):
          s3_client = get_s3_client() # from our aws utils - handles some of the weird localstack stuff
          
          bucket, key = split_s3_url(path)
          file_metadata = s3_client.head_object(Bucket=bucket, Key=key)
          return file_metadata["ContentLength"]          


     file_stats = os.stat(path)
     return file_stats.st_size
```